components/elements/frontPage/LoginForm.js

```diff
@@ -55,6 +62,7 @@ const LoginForm = ({ className, logo }) => {
 
   return (
     <div className="container_login_form">
+      {!state.isForceChangePassword ? <>
```

Comment:
Instead of using the `?:` operator, can we do two blocks instead?

```
{!state.isForceChangePassword &&(
//... Case 1
)}
{state.isForceChangePassword &&(
//... Case 2
)}
```

I have seen heavily nested `?:` in other codebases and it is a nightmare to understand. 😱